string node_name
bool ready
---